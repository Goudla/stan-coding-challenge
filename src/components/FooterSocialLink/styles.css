.footer-social-link {
  color: #fff;
  flex: 1;
  margin-top: 42px;
  margin-right: 20px;
}

.footer-social-link:hover,
.footer-social-link:focus,
.footer-social-link:active {
  color: #0099ff;
}
