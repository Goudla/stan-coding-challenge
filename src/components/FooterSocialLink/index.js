import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class FooterSocialLink extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <a href="https://www.stan.com.au/" className="footer-social-link">
        {children}
      </a>
    );
  }
}

export default FooterSocialLink;
