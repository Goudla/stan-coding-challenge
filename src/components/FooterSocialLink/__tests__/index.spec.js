import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import FooterSocialLink from '../';

describe('<FooterSocialLink />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<FooterSocialLink>Test</FooterSocialLink>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
