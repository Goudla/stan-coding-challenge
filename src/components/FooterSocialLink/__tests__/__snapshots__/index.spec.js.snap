// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterSocialLink /> Should render 1`] = `
<a
  className="footer-social-link"
  href="https://www.stan.com.au/"
>
  Test
</a>
`;
